src/main/java/org/gridsuite/sensitivityanalysis/server/service/SensitivityAnalysisService.java

```diff
@@ -94,6 +109,42 @@ public List<String> getProviders() {
                 .collect(Collectors.toList());
     }
 
+    public Long getFactorsCount(SensitivityFactorsIdsByGroup factorIds, UUID networkUuid, String variantId, Boolean isInjectionsSet) {
+        Long containersAttributesCount = 1L;
+        if (Boolean.TRUE.equals(isInjectionsSet)) {
```

Comment:
Is there a use case where this boolean is false ? If injections or monitoredBranches are not present I think the returned value for factors should be 0 not 1

Reply:
Answer in off discussion